tests/foreman/cli/test_satellitesync.py

```diff
@@ -404,7 +406,6 @@ def test_positive_export_import_cv(self):
         self.assertTrue(len(imported_packages) > 0)
         self.assertEqual(len(exported_packages), len(imported_packages))
 
-    @skip_if_bug_open('bugzilla', 1655239)
```

Comment:
If the skip is no longer justified I'd suggest just adding the number to the docstring (`:bz: 1655239`) so that we have some history recorded in case something similar reappears

Reply:
Done !